package.json

```diff
@@ -29,20 +29,21 @@
     "html"
   ],
   "devDependencies": {
-    "@markedjs/html-differ": "^2.0.1",
+    "@markedjs/html-differ": "^3.0.0",
+    "acorn": "^7.0.0",
```

Comment:
Is acorn used?

Reply:
I'm not sure where that came from? everything seems to work without it.